src/Autosuggest.vue

```diff
@@ -241,7 +241,11 @@ export default {
       };
     },
     isOpen() {
-      return this.shouldRenderSuggestions(this.totalResults, this.loading)
+      const shouldRender = this.shouldRenderSuggestions(this.totalResults, this.loading);
+      if(shouldRender) {
+        this.$emit('showRenderedSuggestions');
+      } else this.$emit('hideRenderedSuggestions');
+      return shouldRender;
```

Comment:
Thanks for the PR!

1. I think we should simplify the naming of the events to `opened` and `closed` since nothing else will be opening/closing on the component in the future. An aside, event names should always be kebab-cased. https://vuejs.org/v2/guide/components-custom-events.html#Event-Names
2. I also know that vue recommends to not put side effects inside computed props. I'm not sure event emitting is technically a side effect, but might be better to not over-emit the event in the event that it doesn't change. Can you refactor to use a watcher for `isOpen` and diff the oldVal vs. newVal so we only fire open/close when it changes (instead of firing it whenever the property is accessed)? 
3. Prefer to use ternary with a single emit call:
```js
isOpen: {
  handler(newValue, oldValue){
    if (newValue !== oldValue) {
      this.$emit(newValue ? 'opened' : 'closed');      
    }
  }
}
```